### PR TITLE
Sprint3/place schema

### PR DESCRIPTION
**Pull request by Mattioo7**

### CODE REVIEW

File: app/(tabs)/index.tsx [hunk 1]
```diff
@@ -40,15 +41,64 @@ export default function TabOneScreen() {
...
-		} finally {
-			setLoading(false);
+		}
+	};
+
+	const getEventsWithFilters = async (categoryId?: number, searchText?: string, sortBy?: string) => {
+		try {
+			let fetchedEvents;
+			if (categoryId !== undefined) {
+				const headers = {
```
Comment (jacob-b-labs) [inline app/(tabs)/index.tsx:51]: Would be better to call it `const requestParams`.

File: app/(tabs)/index.tsx [hunk 2]
```diff
@@ -61,91 +111,82 @@ export default function TabOneScreen() {
...
-			const reservation = {
-				ok: true,
-				data: {
-					reservationToken: "123",
-					placeId: 456
-				}
-			}
-			// console.log("Fetched getCategories");
+			const headers = {
+				headers: {
```
Comment (jacob-b-labs) [inline app/(tabs)/index.tsx:117]: `requestParams`

File: app/event.tsx [hunk 3]
```diff
@@ -32,63 +53,146 @@ export default function ModalScreen() {
 		}
 	};
 
+	const makeReservation = async (event: Event, placeId: number) => {
```
Comment (jacob-b-labs) [inline app/event.tsx:56]: Wouldn't it be better to extract this function? It seems to be repeated

File: app/(tabs)/index.tsx (hunk 1 above)
Comment (Mattioo7) [reply app/(tabs)/index.tsx:51]: No

File: app/(tabs)/index.tsx (hunk 2 above)
Comment (Mattioo7) [reply app/(tabs)/index.tsx:117]: No

File: app/event.tsx (hunk 3 above)
Comment (Mattioo7) [reply app/event.tsx:56]: No

**Review by jacob-b-labs — APPROVED** [review]

**Pull request merged**